site/templates/nav.jet

```diff
@@ -2,73 +2,67 @@
 {{import "./nav/user_logged_in.jet"}}
 
 {{block nav(showSubNav=true)}}
-<div class="navigation" data-toggle="affix">
-  <nav class="navbar navbar-expand-md navbar-dark">
-      <a class="navbar-brand" href="{{ routeToPath("/") }}">{{i18n("nav_homepage")}}</a>
-      
+<div class="navigation search-hidden" data-toggle="affix" aria-hidden="true">
+  <nav>
+
+    <div class="navbar">
```

Comment:
Isn't this `div` redundant?  Why not do `<nav class="navbar">` like it was before?

Reply:
Fixed.